src/main/java/ee/test_gov/sid/mock/controller/AuthenticationSession.java

```diff
@@ -27,6 +27,12 @@ public class AuthenticationSession {
     public HttpResponse<Map<String, Object>> authenticationSession(@PathVariable String identifier, @Body AuthenticationInitData body) {
         UUID sessionId = UUID.randomUUID();
         log.info("Received request to authenticate '{}' with session id '{}'", identifier, sessionId);
+
+        try {
+            dataProvider.putRequestData(identifier, body);
```

Comment:
No info level logging?

Reply:
If any logging, then debug or trace level, but which and how much I would decide under logging improvements task.